administrate/config/locales/administrate.pt-BR.yml

```diff
@@ -0,0 +1,24 @@
+# encoding: UTF-8
+---
+pt-BR:
+  administrate:
+    actions:
+      confirm: Você tem certeza?
+      destroy: Deletar
+      edit: Editar
+      show: Visualizar
```

Comment:
Below you use `Mostrando`, which looks like a form of `Mostrar` - what's the difference between that and `Visualizar`?